File: app/src/main/java/com/callmangement/EHR/imagepicker/listener/OnImageSelectionListener.java
package com.callmangement.EHR.imagepicker.listener;



import com.callmangement.EHR.imagepicker.model.Image;

import java.util.List;

/**
 * Created by hoanglam on 8/18/17.
 */

public interface OnImageSelectionListener {
    void onSelectionUpdate(List<Image> images);
}

File: app/src/main/java/com/callmangement/ehr/imagepicker/listener/OnImageSelectionListener.kt
package com.callmangement.ehr.imagepicker.listener

import com.callmangement.ehr.imagepicker.model.Image


/**
 * Created by hoanglam on 8/18/17.
 */
interface OnImageSelectionListener {
    fun onSelectionUpdate(images: List<Image?>?)
}
